lib/lbt/bundle/Builder.js

```diff
@@ -36,6 +36,9 @@ function makeStringLiteral(str) {
 		return strReplacements[char];
 	}) + "'";
 }
+function removeHashbangs(str) {
+	return str.replace(/^#!(.*)/g, "");
```

Comment:
```suggestion
	return str.replace(/^#!(.*)/, "");
```

Global flag should not be necessary

Reply:
done